added form factor text and exercises

marcgri opened the pull request:
TO DO: need to separate from Chapter 1. Need to add dispersion explanation.

Commit:
added form factor text and exercises

chuckie82 commented:
Looks good. The units of q is often confusing for beginners (and experts). It would be good to clarify the units.
1) There are some typos. Please fix them.
2) Remove duplicated sections from Chapter 1 (e.g. Caffeine related material)

marcgri commented:
Thanks. I removed the Chapter 1 caffeine sections, fixed some typos, and added a short amount of text about dispersion/absorption. I will clean up a little more and submit for review.

I might have to wait to meet with Zhen on Wed while I look at the conformations and El Nemo more. Generating ideal SPI images by today was ambitious--perhaps get the example nailed down this week?

________________________________
From: chuckie82 <notifications@github.com>
Sent: Monday, April 1, 2019 1:44 PM
To: chuckie82/Tutorial-in-Diffraction-Imaging-Python
Cc: Griesemer, Marc B.; Author
Subject: Re: [chuckie82/Tutorial-in-Diffraction-Imaging-Python] added form factor text and exercises (#1)


Looks good. The units of q is often confusing for beginners (and experts). It would be good to clarify the units.

  1.  There are some typos. Please fix them.
  2.  Remove duplicated sections from Chapter 1 (e.g. Caffeine related material)

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/chuckie82/Tutorial-in-Diffraction-Imaging-Python/pull/1#issuecomment-478740400>, or mute the thread<https://github.com/notifications/unsubscribe-auth/At_jT6Pbx9ok9dilNkC_onUkBPYJI64Oks5vcm-rgaJpZM4cWjVf>.

Commit:
added dispersion section and minor corrections
